src/main/java/org/corfudb/cmdlets/corfu_smrobject.java

```diff
@@ -175,7 +186,19 @@
                     // caused by a disconnection with the remote endpoint.  That kind
                     // of non-determinism is evil.  Just retry a few times via 'for' loop.
                     log.warn("WHOA, 'Disconnected endpoint', looping...\n");
-                    try { Thread.sleep(50); } catch (InterruptedException ie){};
+                    try {
+                        Thread.sleep(50);
+                    } catch (InterruptedException ie) {
+                    }
+                    continue;
+                } else if (e.getCause().getClass() == WrongEpochException.class) {
+                    WrongEpochException we = (WrongEpochException) e.getCause();
+                    LayoutView qq = rt.getLayoutView();
```

Comment:
Please stick to standard names for your variables. qq is not a useful name for a layoutview, which should probably be named lv.